### PR TITLE
[MINOR]pre-built package note's text update

### PR DESCRIPTION
**Pull request by nikitakoselev**
pre-built package note's text has been updated. 
Italic looks much better than round brackets with a "." inside.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To build Spark and its example programs, run:
 ./build/mvn -DskipTests clean package
 ```
 
-(You do not need to do this if you downloaded a pre-built package.)
+*You do not need to do this if you downloaded a pre-built package*.
```
Comment (HyukjinKwon) [inline README.md:32]: hm, not sure. either way doesn't much matter to me. I would just leave it as was.

Comment (nikitakoselev) [reply README.md:32]: Let me disagree. Normally in the English language punctuation comes after the bracket. 
"Something like using other fonts (like Italic), or anything else is better than ("." at the end of the sentence.)."

Comment (srowen) [reply README.md:32]: This does not matter either way, so would leave it, but, punctuation belongs inside the closing paren in a parenthesized sentence. I can't parse your example.

**Review by dongjoon-hyun — COMMENTED** [review]
I also agree with the previous review comments because I'm not sure about this PR's claim.
> Italic looks much better than round brackets with a "." inside.

I'd keep the AS-IS style because it seems not worth to touch.

**Pull request closed without merging**